tests/introspection/tests.py

```diff
@@ -182,6 +182,19 @@ def test_get_indexes_multicol(self):
         self.assertNotIn('first_name', indexes)
         self.assertIn('id', indexes)
 
+    @skipIf(connection.vendor == 'mysql', "MySQL doesn't support column ordering for indexes")
```

Comment:
This should be a feature flag on the database backend (`django/db/backends/base/features.py`) rather than a vendor check.